### PR TITLE
update chatbot ui and added few functionality

### PR DESCRIPTION
**Pull request by IsmaelMours**

### CODE REVIEW

File: css/sections/chatbot.css [hunk 1]
```diff
@@ -83,3 +85,53 @@
   display: flex;
   gap: 10px; /* Adds space between the input field and send button */
 }
+
+/* Bot message styling */
+.bot-message {
+  display: flex;
+  justify-content: flex-end; /* Align bot message to the right */
+  margin-bottom: 10px;
+}
```
Comment (Tepanyega) [inline css/sections/chatbot.css:94]: try to use grid

**Review by Tepanyega — CHANGES_REQUESTED** [review]
please submit before the end of the day.

File: css/sections/chatbot.css (hunk 1 above)
Comment (IsmaelMours) [reply css/sections/chatbot.css:94]: I will work on it

**Review by IsmaelMours — COMMENTED** [review]
Thank you

**Pull request closed without merging**